### ISSUE TITLE
nova:debug routes are not working

### ISSUE THREAD

**Issue by PatrickHurley75** [TelescopeJS/Telescope#1556]
The following shows up periodically on Logs in Galaxy. The app loads slowly and can crash for a few seconds.  Any ideas?

'error while server-rendering' 'Error\n at new ApolloError (/app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:1498:23)\n at /app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2179:39\n at /app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2647:25\n at Array.forEach (native)\n at /app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2644:27\n at Array.forEach (native)\n at QueryManager.broadcastQueries (/app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2641:42)\n at QueryManager.broadcastNewStore (/app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2078:14)\n at /app/bundle/programs/server/npm/node_modules/apollo-client/apollo.umd.js:2687:44\n at Object.dispatch (/app/bundle/programs/server/npm/node_modules/redux/lib/applyMiddleware.js:45:18)\n => awaited here:\n at Function.Promise.await (/app/bundle/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:35:12)\n at Object.preRender (packages/nova:routing/lib/routing-server.jsx:62:22)\n at packages/reactrouter:react-router-ssr/lib/server.jsx:213:23\n at [object Object]._.extend.withValue (packages/meteor.js:1122:17)\n at generateSSRData (packages/reactrouter:react-router-ssr/lib/server.jsx:181:29)\n at sendSSRHtml (packages/reactrouter:react-router-ssr/lib/server.jsx:107:25)\n at packages/reactrouter:react-router-ssr/lib/server.jsx:94:13\n at runWithEnvironment (packages/meteor.js:1176:24)\n at packages/meteor.js:1189:14\n at /app/bundle/programs/server/npm/node_modules/react-router/lib/match.js:67:5'

**Comment by SachaG** [comment]
If you update to the latest `devel`, I've added an extra `console.log` to get more info on this type of error. Maybe that'll help track down the problem?

**Comment by PatrickHurley75** [comment]
I have tried to get to the bottom of this rendering issue myself. It is way beyond me. I did a fresh clean install of devel and ran the program with no customisations at all.   localhost:3000/settings causes the following rendering error. It is very like the error that keeps showing up on my Galaxy container. Not enough of a problem to cause it to crash but it is causing occasional "fetch" errors during navigation on the site.  
I got this from the console as per your update.

{ [Invariant Violation: Objects are not valid as a React child (found: object with keys {title, tagline, logoUrl, logoHeight, logoWidth, faviconUrl, requirePostsApproval, requireViewInvite, requirePostInvite, enableNewsletter, autoSubscribe, emailNotifications, postInterval, RSSLinksPointTo, commentInterval, maxPostsPerDay, startInvitesCount, postsPerPage, language, locale, twitterAccount, facebookPage, googleAnalyticsId, useSegment, segmentWriteKey}). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons.] name: 'Invariant Violation', framesToPop: 1 }
